src/sig/sig.h

```diff
@@ -63,108 +84,123 @@ struct OQS_SIG {
 	size_t length_public_key;
 	/** The (maximum) length, in bytes, of secret keys for this signature scheme. */
 	size_t length_secret_key;
-	/** The (maximum) length, in bytes, of a signature for this signature scheme. */
-	size_t max_length_signature;
+	/** The (maximum) length, in bytes, of signatures for this signature scheme. */
+	size_t length_signature;
```

Comment:
I wonder if we should call this `length_signature_max` instead of `length_signature`, since it's not necessarily the length of the signature but the maximum length of the signature.

Reply:
given that maximum was implied in the comment, and that applied to the keys as well, I didn't want to single the signature field with a special "max" marker in its name

Reply:
Okay, makes sense.